man/systemd-sbsign.xml

```diff
@@ -49,6 +49,20 @@
         <xi:include href="version-info.xml" xpointer="v257"/>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>export-signed-attributes</option></term>
```

Comment:
hmm, wouldn't it be better to name this "prepare-offline-signing" or so?

i mean, it exports the signed attributes, true, but I think we should put the focus on the purpose here. and given that there is no other use for this than preparing offling signing (or is there?) I think we should just call it that.